CHANGELOG.md

```diff
@@ -37,25 +37,118 @@ Ref: https://keepachangelog.com/en/1.0.0/
+
+### Docs
+* (license) [\#170](https://github.com/Finschia/finschia/pull/170) fix license copyright holder and typo
+
+
+## [v1.0.0-rc4](https://github.com/Finschia/finschia/releases/tag/v1.0.0-rc4) - 2023-04-08
+
+### Features
+
+### Improvements
+
+### Bug Fixes
+* (lbm-sdk) [\#167](https://github.com/Finschia/finschia/pull/167) Bump github.com/line/lbm-sdk from v0.47.0-rc3 to v0.47.0-rc4
+
+### Breaking Changes
+
+### Build, CI
+
+### Docs
+
+
+## [v1.0.0-rc3](https://github.com/Finschia/finschia/releases/tag/v1.0.0-rc3) - 2023-04-03
+
+### Bug Fixes
+* (lbm-sdk) [\#162](https://github.com/Finschia/finschia/pull/162) Bump github.com/line/lbm-sdk from v0.47.0-rc2 to v0.47.0-rc3
+
+### Breaking Changes
+* (ibc-go)[\#164](https://github.com/Finschia/finschia/pull/164) bump up ibc-go v3.3.2 for change ibc light client of Ostracon to Tendermint
+
+### Build, CI
+* (ci)[\#163](https://github.com/Finschia/finschia/pull/163) fix `release-build` ci error occurred when adding assets after tagging
+
+### Docs
+* (doc)[\#165](https://github.com/Finschia/finschia/pull/165) update license notice and code_of_conduct
+
+
+## [v1.0.0-rc2](https://github.com/Finschia/finschia/releases/tag/v1.0.0-rc2) - 2023-03-29
+
+### Improvements
+* (x/wasmd) [\#158](https://github.com/Finschia/finschia/pull/158) bump up wasmd version to v0.1.0
+* (lbm-sdk) [\#159](https://github.com/Finschia/finschia/pull/159) Bump github.com/line/lbm-sdk from v0.47.0-rc1 to v0.47.0-rc2
+
+### Build, CI
+* (build) [\#157](https://github.com/Finschia/finschia/pull/157) add build args
+
+### Docs
+* (doc) [\#156](https://github.com/Finschia/finschia/pull/156) modify broken link or typo doc file and add issue and pr template
+
+
+## [v1.0.0-rc1](https://github.com/Finschia/finschia/releases/tag/v1.0.0-rc1) - 2023-03-24
+
+### Features
+* (build) [\#150](https://github.com/Finschia/finschia/pull/150) Modify the Makefile to build release bundles
+* (build) [\#153](https://github.com/Finschia/finschia/pull/153) rename cli name to `fnsad`
+* (lbm-sdk) [\#154](https://github.com/Finschia/finschia/pull/154) Bump github.com/line/lbm-sdk from v0.47.0-alpha1.0.20230214070148-11966d123415 to v0.47.0-rc1
+
+### Improvements
+* (x/wasmd) [\#147](https://github.com/Finschia/finschia/pull/147) update wasmd version
+
+
+## [v1.0.0-rc0](https://github.com/Finschia/finschia/releases/tag/v1.0.0-rc0) - 2023-02-16
+
+### Features
+* (build) [\#126](https://github.com/Finschia/finschia/pull/126) Automatically generates release note and binaries
+* (x/wasmd) [\#129](https://github.com/Finschia/finschia/pull/129) chore: apply detached x/wasmd
+* (build) [\#130](https://github.com/Finschia/finschia/pull/130) Add a release build for the linux/arm64, darwin/amd64, and darwin/arm64 platform
+* (lbm-sdk) [\#137](https://github.com/Finschia/finschia/pull/137) Bump line/lbm-sdk to 6c84a4cffa
```

Comment:
```suggestion
* (lbm-sdk) [\#137](https://github.com/Finschia/finschia/pull/137) Bump Finschia/finschia-sdk to 6c84a4cffa
```

Reply:
Since this is just a commit message, I'd prefer not to modify it.